targets/csharp/templates/PlayFabUtil.cs.ejs

```diff
@@ -111,7 +111,7 @@ namespace PlayFab
             if (!string.IsNullOrEmpty(envFileContent))
             {
                 var jsonPlugin = PluginManager.GetPlugin<ISerializerPlugin>(PluginContract.PlayFab_Serializer);
-                dynamic envJson = jsonPlugin.DeserializeObject(envFileContent);
+                JsonObject envJson = jsonPlugin.DeserializeObject<JsonObject>(envFileContent);
```

Comment:
Please note that I'm running out of options here.
1. `JsonObject` is a class in PlayFab.Json, I'm not sure if any serializer can deserialize to it.
2. Creating the object type to deserialize into would require to use reflection if we wish to get the fields by property name rather than by constraining to fixed type. _Making the whole method not string property based isn't a bad idea but is not consistent with what we've done elsewhere, however I'm open to this option_
3. Using `dynamic` requires dependencies like `Microsoft.CSharp` and `System.Dynamic.Runtime` which we do not have on the SDK.

Reply:
1. It'll only work with one specific serializer. It'll likely outright fail if the customer swaps out the serializer plugin.
3. I agree, dynamic would be bad.
2. Could you not make a strong class definition of the structure of this file? For example, we have a strong and complete definition of what's in TestTitleData.json. Even if you change it, this version of this SDK should know what it's looking for, and deserialize into the static type that it expects. Why does it need to be dynamic?

Reply:
Strong type it is.